### ISSUE TITLE
Utterances not imported from existing file

### ISSUE THREAD

**Issue by sudeepjd** [wit-ai/wit#2151]
**Do you want to request a *feature*, report a *bug*, or ask a *question* about wit?**
Bug

**What is the current behavior?**
When we are creating a new App and importing the exported file. The utterance does not get imported. Intents and entities are imported, but the Utterances are not imported.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem.**
Go to an app.
Go to Setting and Export the date to a Zip file.
Create a new app.
Enter a new name and Select the exported file and click create
Check Utterances, it is empty.

_I then added a new utterance and now, the entire list of created utterances shows up in the new app._

**What is the expected behavior?**
Utterances from the exported file should get imported, when the app is created.

**If applicable, what is the App ID where you are experiencing this issue? If you do not provide this, we cannot help.**
264641695026392

**Comment by kitchWWW** [comment]
Hi! Thanks so much for flagging this for us. We were able to reproduce this, and have added this bug to our backlog. In the mean time if you run into this problem again, you don't actually have to add new utterances, you should be able to see your utterances simply by refreshing the page. As you do seem currently unblocked and your utterances are showing up in your app, I'm going to go ahead and close this. Please feel free to re-open if you're not unblocked, or create a new issue if you encounter a new problem. Thanks!
